index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>ScriptTree</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <div class="header">
+        <h1>Why Kill a Tree on Christmas?</h1>
+    </div>
+    <div class='workspc'>
+        <form action='' name='theForm' method="get">
+            <div class='formSpc1'>
+                <label>Please introduce Pine Height  <input type="number" name="height" maxlength="2" id='heightNumber'></label>
```

Comment:
You can't use `maxlength` with `type="number"`. Try `min`, `max`, and `step`

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input